openassistant/templates/hub.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+
+import datasets
+
+
+@dataclass
+class OpenAssistantConfig(datasets.BuilderConfig):
+    """BuilderConfig for OpenAssistant datasets."""
+
+    name: str = None
+    version: datasets.Version = None
+    description: str = None
+    schema: str = None
+    subset_id: str = None
+
+
+lm_features = datasets.Features(
```

Comment:
These "features" are the schema associated with each dataset. I need to check out `data_schemas.md` to integrate them here as well.

For language modelling, I just picked the schema from The Pile (happy to change it)